### PR TITLE
Documentation for #295

### PR DESCRIPTION
**Pull request by zfrhv**
Added documetation for:
#295: `kubectl get hrq` command

And also corrected a little the files.

@mochizuki875 do you want me to add something else?
Turned out to be shorter than I expected.

### CODE REVIEW

File: docs/user-guide/concepts.md [hunk 1]
```diff
@@ -280,8 +280,7 @@ their members.
 To implement hierarchical quotas, HNC automatically creates `ResourceQuota` objects in each
 affected namespace. This is a part of the internal implementation and shouldn't be modified or
 inspected. Use the `kubectl hns hrq` command to inspect hierarchical quotas,
```
Comment (mochizuki875) [inline docs/user-guide/concepts.md:282]: @zfrhv Could you please add `kubectl get hrq` here too?

**Comment by mochizuki875** [comment]
@zfrhv Thanks!
I leave one comment.
Othere than that, I don't see a problem.

File: docs/user-guide/concepts.md (hunk 1 above)
Comment (zfrhv) [reply docs/user-guide/concepts.md:301]: Done
Tell me if you like it

Comment (mochizuki875) [reply docs/user-guide/concepts.md:301]: @zfrhv 
Sorry, my explanation was inadequate...
I intended tha following fix:

> Use the `kubectl hns hrq` command to inspect hierarchical quotas,

->

Use the `kubectl hns hrq` or `kubectl get hrq` command to inspect hierarchical quotas,

Comment (zfrhv) [reply docs/user-guide/concepts.md:297]: what about now?

Comment (mochizuki875) [reply docs/user-guide/concepts.md:297]: How about fix like that?

https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/163a06afac9978823299425218898f45a34c4946/docs/user-guide/concepts.md?plain=1#L280-L294

-> 

Remove (As written, duplicates quickstart example)

https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/163a06afac9978823299425218898f45a34c4946/docs/user-guide/concepts.md?plain=1#L297

-> 

> Use the `kubectl hns hrq` or `kubectl get hrq` command to inspect hierarchical quotas,

Comment (zfrhv) [reply docs/user-guide/concepts.md:286]: what about now?

Comment (mochizuki875) [reply docs/user-guide/concepts.md:286]: Thanks @zfrhv !
I think it's good.
Let's have @adrianludwin check it out.

**Review by adrianludwin — COMMENTED** [review]
/ok-to-test
/lgtm
/approve

**Pull request merged**